huggingface_sb3/push_to_hub.py

```diff
@@ -283,10 +306,19 @@ def package_to_hub(
     :param local_repo_path: local repository path
     :param video_length: length of the video (in timesteps)
     """
+
+    # Autowrap, so we only have VecEnv afterward
+    if not isinstance(eval_env, VecEnv):
+        eval_env = DummyVecEnv([lambda: eval_env])
+
     msg.info(
-        "This function will save, evaluate, generate a video of your agent, create a model card and push everything to the hub. It might take up to 1min. \n This is a work in progress: If you encounter a bug, please open an issue and use push_to_hub instead."
+        "This function will save, evaluate, generate a video of your agent, "
+        "create a model card and push everything to the hub. "
+        "It might take up to 1min. \n "
+        "This is a work in progress: if you encounter a bug, please open an issue "
+        "and use push_to_hub instead."
     )
-    huggingface_token = HfFolder.get_token()
+    HfFolder.get_token()
```

Comment:
- We can remove `HfFolder.get_token()`

Reply:
Done in https://github.com/huggingface/huggingface_sb3/pull/10/commits/7788b5a1ab63b3eef4cd6a1e1da929b7bdd3a371